neard/src/runtime/mod.rs

```diff
@@ -417,6 +428,11 @@ impl NightshadeRuntime {
             #[cfg(feature = "protocol_feature_evm")]
             evm_chain_id: self.evm_chain_id(),
             profile: Default::default(),
+            migration_context: MigrationContext {
+                is_first_block_with_current_version: current_protocol_version
+                    != prev_block_protocol_version,
+                migration_data: self.migration_data.clone(),
```

Comment:
Some style guides recommend spelling this as `Arc::clone(&self.migration_data)`, to make it clear that we are not deep-cloning large vectors here.